### DISCUSSION TITLE
How to get ssl to work? recompile?

### DISCUSSION THREAD

**Discussion by amelsen** [scandum/tintin#133] [General]
What do I have to do precisely to get #ssl connection to work.. it said to ./configure to recompile... 

Did an apt-get install gnutls-bin first.. did ./configure afterwards... still gives same error?

What to do?

**Comment by prool** [comment]
Hello!

You need libgnutls-dev

For example, in my Debian 11 package is libgnutls28-dev

With best regards,

Serge
